Source/control.cpp

```diff
@@ -13,9 +13,9 @@ namespace devilution {
 
 namespace {
 
-CelOutputBuffer pBtmBuff;
-CelOutputBuffer pLifeBuff;
-CelOutputBuffer pManaBuff;
+	CelOutputBuffer pBtmBuff;

```

Comment:
We should update `.clang-format` to not indent within namespaces I think

Reply:
It should not, I have no idea why it does that. `NamespaceIndentation: None` should be enough. I can't reproduce this behavior [there](https://zed0.co.uk/clang-format-configurator/) either.

Reply:
I don't get some of these rules... they are so incredibly inconsistent. You have indentation for every type of scope in the language, but namespaces are for some reason an exception.

The overall inconsistency of things when it comes to C/C++ is one of its worst aspects to me compared to my native language.

It's basically like they are trying to violate the Principle of Least Astonishment at every opportunity possible.

Reply:
> The overall inconsistency of things when it comes to C/C++

Oh there is nothing like that in C, it's just C++ that's like that

Reply:
There is on in root and one in each src folder, so probably they are not in sync.

The reason for this is that some of the upber header files like enum.h are hand aligned. I guess a saner approach at this point is to just mark these headers with clang-format off. There are less of them now anyway.

Reply:
> The overall inconsistency of things when it comes to C/C++ is one of its worst aspects to me compared to my native language.

The C++ language does not define any style, that is left as an appreciation to the writer. Sure, more recent languages like Golang include that (and that's a good thing), but remember how old C++ is compared to Golang. On the scale of computer science projects C++ is a mammoth while Golang is a small rodent. :wink: 

> It's basically like they are trying to violate the Principle of Least Astonishment at every opportunity possible.

Also remember that C++ chose to be 99% compatible with C, so many of the choices had already been taken, and some of them not so great. Adding new features while staying consistent with existing ones created by other people might not always be that simple. Objective-C on the other hand decided to be 100% compatible with C and, well, let's say that I'm not a big fan of it. :slightly_smiling_face: 

> There is on in root and one in each src folder, so probably they are not in sync.

Ah that explains things.

> The reason for this is that some of the upber header files like enum.h are hand aligned. I guess a saner approach at this point is to just mark these headers with clang-format off. There are less of them now anyway.

:+1:

Reply:
Ok, should at least not be nested any longer, but there are still multiple copies which isn't ideal either.

Reply:
Oh, all the clang-format issues are now resolved.